README.md

```diff
@@ -1,77 +1,373 @@
 
+You can use GitHub to build software alongside millions of other developers. You can also use it to learn how to code, or find help with programming problems.
 
-[Github Tutorial](https://github.com/jlord/git-it-electron#what-to-install) (don't worry too much about reading all of these. Wet your pallette and bookmark for later)
+GitHub offers an easy way to share your code with friends and classmates, or collaborate on open-source projects.
 
-### Browser
 
-[Download Chrome](https://www.google.com/intl/en_ca/chrome/) (This is our browser we prefer for our dev environment.)
+#### [Dev.to](https://dev.to/)
 
+If you're a developer looking for a career change or an aspiring developer who wants to learn more, Dev.to is the place to start.
 
-<h2 id="html">HTML & CSS</h2>
+Dev.to is an online community where developers from all backgrounds can share their experiences and expertise. It's also the perfect place for anyone interested in learning about software development to connect with like-minded individuals working in the field.
+
+You can find jobs on Dev.to, as well as connect with recruiters and companies seeking employees with your skillset. You can also learn about new technologies, get advice on making the most of your experience and career path, and share tips on success in this industry.
+
+Whether you're looking for a new job or want to meet other developers, Dev.to is an essential part of any aspiring developer's toolkit!
+ 
+At #VetsWhoCode We Believe that long-form writing is one of the best ways to scale yourself, so you don't have to retain information, showcase how you communicate, and a simple way to advocate for yourself to non-technical employers when you're not writing since everyone on here is in tech so the people will be your audience.
+
+### Command Line
+
+The command line is a text-based interface for interacting with your computer. It's where you can type in commands to perform tasks like copying files and managing folders and running applications.
+ 
+It's often called the "command prompt" or "terminal," but it's also known as the "console" or "CLI," which stands for "command line interface." The application name used to access the command line differs from OS to OS: On Windows, it's called cmd (for Command Prompt), while on Mac, it's called Terminal.
+
+Following will be the directions to operate in the command line for MacOS and Windows.
+
+### Command Line Operations 
+
+#### Open
+You can use the Terminal app to access your computer's command line. The Terminal is one of the most powerful tools on your Mac, and it's how we'll be running our code.
+
+MacOS: To open Terminal, go to Applications, then to Utilities finally to Terminal.
+Windows: To open the command prompt, click Start and then click All Programs. In the programs list, click Accessories. In the accessories list, click Command Prompt.
```

Comment:
If we're going to have a Command Line tutorial, I'd assume we'd want 2 drop downs. 1 drop down for the tutorial with Windows and 1 for the tutorial for Mac.